controllers/apply/form-router-next/views/delete.njk

```diff
@@ -5,19 +5,25 @@
 {% block content %}
     <main role="main" id="content">
         <div class="content-box u-inner u-inner-wide-only">
-            <div class="s-prose u-constrained-wide">
-                <h1>Are you sure you want to delete your application?</h1>
-
-                <h2 class="t2 t--underline">This can't be undone.</h2>
-                <p>If you're sure you want to delete your application, then you can proceed below &ndash; please note this this cannot be recovered.</p>
+            <div class="form-header">
```

Comment:
Worth adding the breadcrumbs in here too as another navigation option?
![image](https://user-images.githubusercontent.com/394376/59028326-4272e580-8853-11e9-8f7a-39f2f5f12767.png)